Sources/Purchasing/StoreKit2/StoreKit2TransactionFetcher.swift

```diff
@@ -307,11 +307,22 @@ extension StoreKit2TransactionFetcher {
                     return nil
                 }
             } catch {
-                self.diagnosticsTracker?.trackAppleAppTransactionError(errorMessage: error.localizedDescription)
-                Logger.warn(Strings.storeKit.sk2_error_fetching_app_transaction(error))
+                trackAppleAppTransactionError(error)
```

Comment:
Should we add a `self` here?
```suggestion
                self.trackAppleAppTransactionError(error)
```

Reply:
Added explicit selfs 👍